assets/js/components/wp-dashboard/wp-dashboard-modules.js

```diff
@@ -36,10 +37,10 @@ class WPDashboardModules extends Component {
 	render() {
 		return (
 			<Fragment>
-				<div className={ `
-					googlesitekit-wp-dashboard-stats
-					${ global.googlesitekit.modules.analytics && global.googlesitekit.modules.analytics.active ? 'googlesitekit-wp-dashboard-stats--fourup' : '' }
-				` }>
+				<div className={ classnames(
+					'googlesitekit-wp-dashboard-stats',
+					{ 'googlesitekit-wp-dashboard-stats--fourup': global.googlesitekit.modules.analytics && global.googlesitekit.modules.analytics.active }
```

Comment:
We can clean up this conditional a little bit as `analytics` will always be set.
```suggestion
					{ 'googlesitekit-wp-dashboard-stats--fourup': global.googlesitekit.modules.analytics.active }
```